scenarios/har/1/openapi.yaml

```diff
@@ -610,23 +601,18 @@ externalDocs:
 components:
   schemas:
     Order:
-      x-swagger-router-model: io.swagger.petstore.model.Order
```

Comment:
Is this something we want to support? It's not part of the schema ajv understands out of the box. 

It's not hard too add these but I'm not entirerly sure how common they are in the wild!